src/sentry/web/urls.py

```diff
@@ -67,6 +67,14 @@
         )
     ]
 
+# demo mode
+if settings.DEMO_MODE:
```

Comment:
@mattrobenolt ok, I added the check here so we don't load expose the route at all if `DEMO_MODE` is `False`.